rules/prefer_to_have_length.js

```diff
@@ -17,7 +17,11 @@ module.exports = context => {
         const argumentObject = node.arguments[0].object;
         const argumentProperty = node.arguments[0].property;
 
-        if (propertyName === 'toBe' && argumentProperty.name === 'length') {
+        if (
+          argumentProperty &&
+          propertyName === 'toBe' &&
```

Comment:
just a nit, can we switch these two lines?

Reply:
what do you mean?

Reply:
Oh, gotcha

Reply:
Sorry for being indirect :D